crates/cast/src/helpers/scarb_utils.rs

```diff
@@ -155,39 +156,21 @@ pub fn get_package_metadata<'a>(
 
 pub fn parse_scarb_config(
     profile: &Option<String>,
-    path: &Option<Utf8PathBuf>,
+    package_metadata: Option<&PackageMetadata>,
 ) -> Result<CastConfig> {
```

Comment:
let's rename this function, maybe `get_cast_config`?

Reply:
Done